[2022-03-15 08:15:29,368] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: glue_poc.create_bucket scheduled__2022-03-13T00:00:00+00:00 [queued]>
[2022-03-15 08:15:29,384] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: glue_poc.create_bucket scheduled__2022-03-13T00:00:00+00:00 [queued]>
[2022-03-15 08:15:29,385] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-03-15 08:15:29,386] {taskinstance.py:1244} INFO - Starting attempt 17 of 17
[2022-03-15 08:15:29,387] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-03-15 08:15:29,402] {taskinstance.py:1264} INFO - Executing <Task(PythonOperator): create_bucket> on 2022-03-13 00:00:00+00:00
[2022-03-15 08:15:29,409] {standard_task_runner.py:52} INFO - Started process 340 to run task
[2022-03-15 08:15:29,414] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'glue_poc', 'create_bucket', 'scheduled__2022-03-13T00:00:00+00:00', '--job-id', '227', '--raw', '--subdir', 'DAGS_FOLDER/glue_poc.py', '--cfg-path', '/tmp/tmplp2criyz', '--error-file', '/tmp/tmpp49fbdu6']
[2022-03-15 08:15:29,416] {standard_task_runner.py:77} INFO - Job 227: Subtask create_bucket
[2022-03-15 08:15:29,488] {logging_mixin.py:109} INFO - Running <TaskInstance: glue_poc.create_bucket scheduled__2022-03-13T00:00:00+00:00 [running]> on host d3c9803ba696
[2022-03-15 08:15:29,568] {taskinstance.py:1431} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=glue_poc
AIRFLOW_CTX_TASK_ID=create_bucket
AIRFLOW_CTX_EXECUTION_DATE=2022-03-13T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-03-13T00:00:00+00:00
[2022-03-15 08:15:29,581] {base_aws.py:194} INFO - Credentials retrieved from extra_config
[2022-03-15 08:15:31,063] {python.py:175} INFO - Done. Returned value was: None
[2022-03-15 08:15:31,096] {taskinstance.py:1282} INFO - Marking task as SUCCESS. dag_id=glue_poc, task_id=create_bucket, execution_date=20220313T000000, start_date=20220315T081529, end_date=20220315T081531
[2022-03-15 08:15:31,284] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-03-15 08:15:31,433] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
